blocklib/analog/noise_source/noise_source_cpu.cc

```diff
@@ -38,8 +38,8 @@ work_return_t noise_source_cpu<T>::work(work_io& wio)
     auto out = wio.outputs()[0].items<T>();
     auto noutput_items = wio.outputs()[0].n_items;
 
-    auto type = pmtf::get_as<int>(*this->param_type);
-    auto ampl = pmtf::get_as<float>(*this->param_amplitude);
+    auto type = pmtv::cast<int>(*this->param_type);
```

Comment:
I would probably use `std::get<int>` unless you have a reason to use cast (here and in all of the other instances of `pmtv::cast`.

Reply:
there are some instances where pmtv::cast would be appropriate, but more in the case where a block accepts a pmt through a message handler or tag, and it is equally sufficient to provide int, long, etc.  For this case, you are right, the type is fixed through the autogen'd code

Reply:
I agree.  A cast is fine when passing in a pmt.